packages/synthetics-sdk-broken-links/package.json

```diff
@@ -13,7 +13,6 @@
     "pretest": "npm run compile",
     "lint": "eslint **/*.ts",
     "posttest": "npm run lint",
-    "postinstall": "node ../../node_modules/puppeteer/install.js",
```

Comment:
Why dont we need this anymore?

Reply:
The package structure of puppeteer changed at some point along the way and now this is no longer required / the file does not even exist anymore.